stable/nextcloud/README.md

```diff
@@ -1,4 +1,9 @@
-# nextcloud
+# DEPRECATED - nextcloud
+
+|**This chart has been deprecated and moved to its new home:**
```

Comment:
Also please add to NOTES.txt (see [Deprecating A Chart](https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart))

Reply:
... and done!